web/src/Main.elm

```diff
@@ -233,33 +233,28 @@ navigation : Html.Html msg
 navigation =
     let
         baseUrl =
-            "https://concourse-ci.org/"
+            "https://concourse-ci.org"
 
         blogUrl =
             "https://blog.concourse-ci.org"
 
         discussUrl =
             "https://discuss.concourse-ci.org"
     in
-    div [ class "navigation" ]
-        [ div [ class "top-logo" ]
-            [ a [ href baseUrl, class "top-link" ]
+    div [ class "page-top" ]
+        [ nav
+            [ class "top-nav" ]
+            [ a [ href <| baseUrl, class "top-link logo-link" ]
                 [ img
-                    [ src <| baseUrl ++ "images/logo-white.svg" ]
+                    [ src <| baseUrl ++ "/images/logo-white.svg" ]
                     []
                 , Html.text "Concourse"
                 ]
-            ]
-        , nav
-            [ class "top-nav" ]
-            [ a [ href <| baseUrl ++ "docs", class "top-link" ] [ Html.text "docs" ]
-            , a [ href <| baseUrl ++ "examples", class "top-link" ] [ Html.text "examples" ]
-            , a [ href <| baseUrl ++ "project", class "top-link" ] [ Html.text "project" ]
+            , a [ href <| baseUrl ++ "/docs.html", class "top-link" ] [ Html.text "docs" ]
+            , a [ href <| baseUrl ++ "/examples.html", class "top-link" ] [ Html.text "examples" ]
+            , a [ href <| baseUrl ++ "/project.html", class "top-link" ] [ Html.text "project" ]
             , a [ href <| blogUrl, class "top-link" ] [ Html.text "blog" ]
             , a [ href <| discussUrl, class "top-link" ] [ Html.text "discuss" ]
             , a [ href "/", class "top-link active" ] [ Html.text "resource types" ]
```

Comment:
there should be a commit that moves this nav item before blog and discuss - maybe pull?

Reply:
resource types gets a higher billing than the blog? we've made the big time!

Reply:
whoops, revived an older version of the branch